### PR TITLE
List page icon was missing in local dev

### LINKED ISSUE #922: List page icon is missing on the pages page and right side bar
To reproduce, on the dev branch cd into src/client and do a `yarn start`
On the pages page, notice that the list page does not have an icon
image

On the right sidebar, notice that the list page is missing the icon:
image

### PR DESCRIPTION
**Pull request by tejbirsingh**
Before:
image

After:
image

This closes #922

### CODE REVIEW

**Comment by tejbirsingh** [comment]
I'm not sure the correct way to link to issues in github like Azure DevOps so I just referenced it #922

**Comment by SahilTara** [comment]
You just need to reference really to link. You can even write "closes #<issue number>" to close the issue when this is merged. Do you know whats up with those plus and minus icons though they seem to be running away from the circle.

**Comment by tejbirsingh** [comment]
I didn't even notice the + and - signs 😲. I can take a look but I would say we should create another issue for it. Then this one keeps a small scope of just the icon.

File: src/client/src/mockData/mockVsCodeApi.ts [hunk 1]
```diff
@@ -272,7 +272,7 @@ const mockVsCodeApi = () => ({
                     requiredVisualStudioWorkloads: []
                   },
                   {
-                    emplateId: "wts.Page.React.List",
+                    templateId: "wts.Page.React.List",
```
Comment (piotrmark) [inline src/client/src/mockData/mockVsCodeApi.ts:275]: ❤

**Review by piotrmark — APPROVED** [review]

**Comment by piotrmark** [comment]
> I didn't even notice the + and - signs 😲. I can take a look but I would say we should create another issue for it. Then this one keeps a small scope of just the icon.

It looks good on my machine :D

image

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by SahilTara — APPROVED** [review]

**Review by jasmingoh — APPROVED** [review]

**Pull request merged**